uint8 seq
int16[] wheel_angular_velocity
float32[] wheel_angle
uint32[] m_bus_update_count
geometry_msgs/Vector3 imu_angular_velocity
geometry_msgs/Vector3 imu_linear_acceleration
geometry_msgs/Vector3 imu_mag_xy
geometry_msgs/Vector3 imu_rpy

